contrib/autocomplete/bash_autocomplete

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
```

Comment:
missing file header

Reply:
"fixing" in containerd/project#39 :) I don't think we should be adding our header and copyright to files we didn't author